pkg/cmd/serve.go

```diff
@@ -95,6 +95,11 @@ func RegisterServeFlags(cmd *cobra.Command, config *server.Config) {
 	cmd.Flags().Uint16Var(&config.MaximumUpdatesPerWrite, "write-relationships-max-updates-per-call", 1000, "maximum number of updates allowed for WriteRelationships calls")
 	cmd.Flags().Uint16Var(&config.MaximumPreconditionCount, "update-relationships-max-preconditions-per-call", 1000, "maximum number of preconditions allowed for WriteRelationships and DeleteRelationships calls")
 
+	cmd.Flags().BoolVar(&config.V1SchemaAdditiveOnly, "for-testing-v1-schema-additive-only", false, "changes V1 schema API to be additive only")
+	if err := cmd.Flags().MarkHidden("for-testing-v1-schema-additive-only"); err != nil {
```

Comment:
I think this is slightly more descriptive:

`testing-only-schema-additive-writes`
"append new definitions to the existing schema, rather than overwriting it"